src/stylesheets/components/_hints.scss

```diff
@@ -34,5 +34,6 @@
   padding: 0 $base-spacing-unit;
 }
 .url {
+  word-wrap: break-word;
```

Comment:
This does not work if applied to a search results url with a long url.

`word-break: break-all` will work